src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -555,11 +555,22 @@ export class ResultsViewPageStore {
             : [];
     }
 
-    @computed get treatmentList() {
-        return this.urlWrapper.query.treatment_list &&
-            this.urlWrapper.query.treatment_list.trim().length
-            ? this.urlWrapper.query.treatment_list.trim().split(/;/)
+    @computed get parsedGenericAssayGroups() {
+        // empty if no heatmap tracks param
+        const groups = this.urlWrapper.query.generic_assay_groups
```

Comment:
we should make this a parsing helper function and have a test for it.

Reply:
Sure!